tests/resources/test_resources_oai.py

```diff
@@ -0,0 +1,232 @@
+# -*- coding: utf-8 -*-
```

Comment:
- Managed sets are not implemented yet, as this has not been discussed properly afaik (adding/removing records manually, managed flag not present)
  - https://github.com/inveniosoftware/invenio-oaiserver/issues/211

- Since the indexing happens automatically after inserting the set in the db through signals, the only way to check if the `search_pattern` is valid or not is to try indexing before (and then it gets reindexed again afterwards). Though this should be fixed by
  - https://github.com/inveniosoftware/invenio-oaiserver/issues/215